README.md

```diff
@@ -32,3 +32,4 @@
 * [ReRoto](https://reroto.com)
 * [wellpaid.io](https://wellpaid.io)
 * [jumpcomedy.com](https://www.jumpcomedy.com)
+* [PartyRentalPlatform](https://partyrentalplatform.com/) - As a website builder. [CentralArkansasInflatables](https://centralarkansasinflatables.com/) for example.
```

Comment:
I'm happy for context here, but suggest we conform to the structure used in other sections, and remove the secon link. I also think you could use the slogan from your website.

```suggestion
* [PartyRentalPlatform](https://partyrentalplatform.com/): All-in-one event rental software.
```